elastalert/ruletypes.py

```diff
@@ -1030,7 +1030,15 @@ class MetricAggregationRule(BaseAggregationRule):
     def __init__(self, *args):
         super(MetricAggregationRule, self).__init__(*args)
         self.ts_field = self.rules.get('timestamp_field', '@timestamp')
-        if 'max_threshold' not in self.rules and 'min_threshold' not in self.rules:
+
+        threshold_keys = [x for x in self.rules
+                          if (x.endswith('_threshold') and x not in ['min_threshold','max_threshold'])
+                          ]
+        thresholds = [(x[:-10], self.rules.get(x)[1:], self.rules.get(x)[:1]) for x in threshold_keys]
+        self.thresholds_above = sorted([(x[0], int(x[1]),) for x in thresholds if x[2] == '>'], key=lambda tup: tup[1])
```

Comment:
Makes sense to use `float`, otherwise we can't have thresholds on 0..1 range metrics. Load averages etc.